Sources/SwiftDocC/Model/Rendering/RenderNode/RenderMetadata.swift

```diff
@@ -144,6 +144,15 @@ public struct RenderMetadata: VariantContainer {
     /// The variants for the source file URI of a page.
     public var sourceFileURIVariants: VariantCollection<String?> = .init(defaultValue: nil)
     
+    /// The source location of the topic's source code.
+    public var source: Source? {
```

Comment:
I left a comment below wondering if we should call this `source`, but regardless of what general name we land on here – should we specify that this is a remote location? That way it's clear how this is different from the existing `sourceFileURI`.

```suggestion
    public var remoteSource: Source? {
```

```suggestion
    public var remoteLocation: Location? {
```

```suggestion
    public var remoteSourceLocation: SourceLocation? {
```